omnibus/.kitchen.yml

```diff
@@ -2,35 +2,89 @@ driver:
   name: vagrant
   forward_agent: yes
   customize:
-    cpus: 2
-    memory: 2048
+    cpus: 4
+    memory: 4096
   synced_folders:
     - ['..', '/home/vagrant/inspec']
+    - ['../../omnibus', '/home/vagrant/omnibus']
```

Comment:
Can do :).  It doesn't affect me much - I already have to mount my entire code directory for the windows platforms below because mounting just inspec won't work on windows (git hates dealing with a network share as a top-level directory).